composer.json

```diff
@@ -20,7 +20,6 @@
         "symfony/framework-bundle": "~2.5|~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.8",
```

Comment:
I'll to wait a few days to see if the compatibility issue between PHPUnit and Symfony is fixed.
I'd like to avoid modifying the dependencies of this project just because of a temporary (I hope) compatibility issue :)